functions/src/index.ts

```diff
@@ -10,20 +10,20 @@ admin.initializeApp({
 
 const db = admin.firestore();
 const gyms = ['teagle', 'helen_newman', 'noyes', 'appel'];
+const round30 = true;
 
 exports.getURL = functions.https.onCall((data: { id: string, startDate: string, endDate: string, offset: number }) => {
-    if (!data.id || !data.startDate || !data.endDate) {
+    if (!data.id || !data.startDate || !data.endDate || !data.offset) {
         throw new functions.https.HttpsError('invalid-argument', 'ID missing!');
     }
-    const startDate = new Date(data.startDate);
-    const endDate = new Date(data.endDate);
     const id = data.id;
-    return getData(id, startDate, endDate, data.offset);
+    const startDate = new Date(new Date(data.startDate));
+    const endDate = new Date(new Date(data.endDate));
```

Comment:
what's the point of this?

Reply:
i believe it's a mistake, I will change it back to what it was before